protostar/starknet/cheatable_cached_state.py

```diff
@@ -24,6 +26,7 @@ def __init__(self, *args: Any, **kwargs: Any):
         self.class_hash_to_contract_abi_map: Dict[ClassHashType, AbiType] = {}
         self.class_hash_to_contract_path_map: Dict[ClassHashType, Path] = {}
         self.contract_address_to_class_hash_map: Dict[AddressType, ClassHashType] = {}
+        self.contract_calls: dict[AddressType, list[CallData]] = {}
```

Comment:
[Functionality] The mapping should include function name `hash(contract_address, fn_name) -> CairoOrPythonData`.

Reply:
[add function name as argument](https://github.com/software-mansion/protostar/pull/1060/commits/03936e2d2755036d14f9f9de05babd3eb807faf8)